src/index.html

```diff
@@ -6,8 +6,291 @@
+                  <a href="#testimonials" class="nav__link">TESTIMONIALS</a>
+                </li>
+                <li class="nav__item">
+                  <a href="#contact" class="nav__link">CONTACTS</a>
+                </li>
+                <a class="nav__call-us" href="tel:+12345555555">
+                  <div class="nav__number">
+                    +1 234 555-55-55
+                  </div>
+                  <div class="nav__call-label">
+                    CALL TO ORDER
+                  </div>
+                </a>
+              </ul>
+            </div>
+          </nav>
+
+          <a class="header__logo" href="#">
+            <img class="header__logo" src="./images/logo.svg"
+              alt="Condo&nbsp;Kings logo">
+          </a>
+
+          <a class="header__phone" href="tel:+12345555555">
+            <img src="./images/phone.svg" alt="Phone logo">
+          </a>
+        </div>
+
+        <h1 class="header__title">
+          Miami Condo&nbsp;Kings
+        </h1>
+      </div>
+    </header>
+    <main class="main">
+      <div class="main__content">
+        <section class="about" id="about">
+          <div class="about__content">
+            <h2 class="about__title">Miami best<br>Real&nbsp;estate&nbsp;<a
+                class="about__link" href="#">here</a>
+            </h2>
+            <div class="about__text">
+              <h3 class="about__small-title">Over 50 years experience in love.
+              </h3>
+              <p class="about__paragraph">
+                Mazel and VA Tsukkerman in his "Analysis
+                of musical works." Pointillism, which originated in the music of
+                the early twentieth century, microforms, found a distant
+                historical.
+                <br>
+                <br>
+                The market leader in real estate since 2003. Awards Real Estate
+                Company, 2010, 2011, 2013. More than 200 satisfied customers
+                premium segment. We focus only on new buildings and high-level
+                doskanalno know this area better than other specialists.Working
+                directly with developers - so have more useful information about
+                the profitable deals.
+                <br>
+                <br>
+                <strong>
+                  We will never be late!
+                  <br>
+                  Guaranteed secure transaction - 100%
+                </strong>
+              </p>
+            </div>
+          </div>
+        </section>
+        <section class="gallery" id="gallery">
+          <div class="gallery__content">
+            <h2 class="gallery__title">Gallery</h2>
+            <img class="gallery__logo" src="./images/crown.svg"
+              alt="Crown logo">
+            <div class="gallery__photos">
+              <div class="gallery__row-1">
+                <img class="gallery__photo photo-1" src="./images/photos/1.jpg"
+                  alt="photo-1">
```

Comment:
```suggestion
                <img 
                  class="gallery__photo photo-1" 
                  src="./images/photos/1.jpg"
                  alt="photo-1"
                >
```

с другими тегами аналогично